tests/console/yast2_dns_server.pm

```diff
@@ -25,10 +25,10 @@ sub assert_running() {
     my $running = shift;
 
     if ($running) {
-        assert_script_run 'systemctl is-active named | grep -E "^active"';
+        assert_script_run '(systemctl is-active named || true) | grep -E "^active"';
     }
     else {
-        assert_script_run 'systemctl is-active named | grep -E "^(inactive|unknown)"';
+        assert_script_run '(systemctl is-active named || true) | grep -E "^(inactive|unknown)"';
```

Comment:
Because the service might be *either* inactive *or* unknown and we want to distinguish. With the grep it's showing what it found. That is how the test was handling it in before and I want to preserve that.